eta/core/models.py

```diff
@@ -927,13 +926,7 @@ class PickledModel(PublishedModel):
     '''Class that can load a published model stored as a .pkl file.'''
 
     def _load(self):
-        try:
```

Comment:
@yashbhalgat note that I moved this `try-except` to a `eta.core.serial.read_pickle`. The idea is that we only want to implement reading pickle files once and then re-use it everywhere. You fixed this case, but there was a duplicate implementation of pickle reading in `Picklable.from_pickle`, so I fixed that.

Reply:
Good idea!